src/consensus/pbft/libbyz/Meta_data_d.cpp

```diff
@@ -108,10 +108,7 @@ bool Meta_data_d::verify()
   std::shared_ptr<Principal> p = node->get_principal(id());
```

Comment:
Comment still says "Check Mac"

Reply:
Also, I believe that all this code can be removed and return `true` directly?

Reply:
this is still needed to validate that the message is coming from a replica and not a client and still needed for a the standalone tests. This in time we be reduced and removed